client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -633,6 +628,8 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 		<div className="pattern-assembler__wrapper" ref={ wrapperRef } tabIndex={ -1 }>
 			{ noticeUI }
 			<div className="pattern-assembler__sidebar">
+				<QueryActiveTheme siteId={ site?.ID } />
```

Comment:
cc: @arthur791004 

Re: https://github.com/Automattic/wp-calypso/pull/85410#discussion_r1432422810, I moved the query here so that the active theme is requested as early as possible, minimizing the chance of flashing content.